File: JavaScriptSolutions/projecteulerProblem2.js
var sum = 0;

var previousTerm = 1;
var currentTerm = 2;

while(currentTerm < 4000000)
{
	let temp = previousTerm;
	previousTerm = currentTerm;
	if(currentTerm % 2 == 0)
	{
		sum += currentTerm;
	}
       	currentTerm = currentTerm + temp;  
}
console.log(sum)

File: PythonSolutions/projecteulerProblem2.py
previousTerm = 1
currentTerm = 2
sum = 0

while(currentTerm <4000000):
    temp = previousTerm
    previousTerm = currentTerm
    if(currentTerm % 2 == 0):
        sum += currentTerm
    currentTerm += temp

print(sum)
